src/akamai/js/akamaiHD_flash.js

```diff
@@ -153,6 +153,13 @@ require("../../../html5-common/js/utils/constants.js");
     var videoItem = video;
     var currentUrl = '';
     var loaded = false;
+    var urlChanged = false;
+    var self=this;
+    var currentTime;
+    var buffer;
+    var totalTime;
+    var seekRange_start;
+    var seekRange_end;
 
```

Comment:
Can we declare the controller property here so it's clear that it exists? Currently it's only referenced inside the constructor and as self.controller.